tests/test_clip_service.py

```diff
@@ -189,50 +196,80 @@ def test_clip_performance_benchmark(self, service_url: str, sample_image_url: st
         
         assert avg_response_time < 10.0
     
-    def test_cat_vs_dog_classification(self, service_url: str, cat_image_path: str, cat_vs_dog_labels: list):
-        """Test that CLIP correctly identifies cat images as cats rather than dogs."""
+    def test_cat_vs_dog_classification(self, service_url: str, cat_image_path: str, dog_image_path: str, cat_vs_dog_labels: list):
+        """Test that CLIP correctly identifies cat images as cats rather than dogs using local sample images."""
         if not os.path.exists(cat_image_path):
             pytest.skip(f"Cat sample image not found at {cat_image_path}")
+        if not os.path.exists(dog_image_path):
+            pytest.skip(f"Dog sample image not found at {dog_image_path}")
         
-        public_cat_url = "https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/transformers/tasks/cat.jpg"
-        
-        payload = {
-            "image_url": public_cat_url,
+        cat_payload = {
+            "image_url": f"file://{cat_image_path}",
             "labels": cat_vs_dog_labels
         }
         
         try:
-            response = requests.post(service_url, json=payload, timeout=30)
-            assert response.status_code == 200
+            cat_response = requests.post(service_url, json=cat_payload, timeout=30)
+            if cat_response.status_code != 200:
```

Comment:
This all looks perfect. Please take out the hugging face image fallback for the cat, and similarly remove the unsplash fallback for the dog